maestro-client/src/main/java/maestro/Errors.kt

```diff
@@ -31,6 +31,8 @@ sealed class MaestroException(override val message: String) : RuntimeException(m
 
     class AppCrash(message: String): MaestroException(message)
 
+    class DriverTimeout(message: String): MaestroException(message)
```

Comment:
I see your point, however, I don't see how these relate. We can map a specific Maestro Exception to not be client-error.

Reply:
There is a risk then right to confidently say it's a client exception. If we really see this as client exception what are those cases and what is client expected to do in that case to fix it?

Reply:
Sorry if I was not clear.
- this will be an infra error and also a MestroException. 
- These 2 things are not tied - They can be today, but I can untie that

Reply:
Ohh now I understood! But this will always be mapped as infra error on cloud right?